### ISSUE TITLE
Jacobian broken since 0.9.15

### ISSUE THREAD

**Issue by baroobob** [pbrod/numdifftools#25]
I wrote some code back in September that uses Jacobian to calculate all the partial derivatives for a matrix valued function of a vector argument. I started working with this code again this morning and discovered that it no longer works. I traced it down to a change in the behavior of Jacobian. This simple program demonstrates what I mean.
```python
from numpy import array
from numdifftools import Jacobian
x = array([1,2])
G = lambda x: array([[x[0], x[1]], [x[0], x[1]]])
dGdx = Jacobian(lambda x: G(x))
D = dGdx(x)
print G(x).shape
print D.shape
```
This is the output with numdifftools 0.9.14 installed.
```
(2, 2)
(2, 2, 2)
```
This is the output with numdifftools 0.9.15 or later installed.
 ```
(2, 2)
(2, 2, 4)
```
So numdifftools 0.9.15 and later returns the wrong size tensor.

**Comment by pbrod** [comment]
Thanks for letting me know. I will look into it.

**Comment by baroobob** [comment]
Using git bisect I was able to narrow it down to this commit:
[4960b5a80c5632396921617e94de4d392681933a] Fixed isssue #19 multiple observations mishandled in Jacobian

**Comment by pbrod** [comment]
Issue fixed in commit 3baecb77180f79242cdab37b084f28a9c4220f28,
Closing the issue.

**Comment by baroobob** [comment]
Wow, that was fast. Thank you!
